app/Models/Image.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Image extends Model
+{
+    protected $table = 'images';
+
+    protected $guarded = ['id'];
+
+    public function users()
+    {
+        return $this->hasMany(\App\Models\User::class, 'image_id');
```

Comment:
If I'm not mistaken, Laravel looks on its own for 'image_id' in the user-table, because the hasMany-Relationship is defined in the image-model. So you could remove 'image_id' here.